sinch/domains/voice/endpoints/calls/get_call.py

```diff
@@ -0,0 +1,48 @@
+from sinch.core.deserializers import timestamp_to_datetime_in_utc_deserializer
+from sinch.core.models.http_response import HTTPResponse
+from sinch.domains.voice.endpoints.voice_endpoint import VoiceEndpoint
+from sinch.core.enums import HTTPAuthentication, HTTPMethods
+from sinch.domains.voice.models.calls.responses import GetVoiceCallResponse
+from sinch.domains.voice.models.calls.requests import GetVoiceCallRequest
+from sinch.domains.voice.models import Price, Destination
+
+
+class GetCallEndpoint(VoiceEndpoint):
+    ENDPOINT_URL = "{origin}/calling/v1/calls/id/{call_id}"
+    HTTP_METHOD = HTTPMethods.GET.value
+    HTTP_AUTHENTICATION = HTTPAuthentication.SIGNED.value
+
+    def __init__(self, request_data: GetVoiceCallRequest):
+        self.request_data = request_data
+
+    def build_url(self, sinch) -> str:
+        return self.ENDPOINT_URL.format(
+            origin=sinch.configuration.voice_origin,
+            call_id=self.request_data.call_id
+        )
+
+    def handle_response(self, response: HTTPResponse) -> GetVoiceCallResponse:
+        super().handle_response(response)
+        return GetVoiceCallResponse(
+            from_=response.body.get("from"),
```

Comment:
https://developers.sinch.com/docs/voice/api-reference/voice/tag/Calls/#tag/Calls/operation/Calling_GetCallResult!c=200&path=from&t=response it seems from is also of Destination type

Reply:
Good catch 👌 fixed